install/kubernetes/cilium/values.yaml.tmpl

```
@@ -656,13 +656,13 @@ hubble:
 
   # -- Buffer size of the channel Hubble uses to receive monitor events. If this
   # value is not set, the queue size is set to the default monitor queue size.
-  # eventQueueSize: ""
+  eventQueueSize: ""
```

just to avoid parsing error

```suggestion
  # -- Buffer size of the channel Hubble uses to receive monitor events. If this
  # value is not set or zero, the queue size is set to the default monitor queue size.
  eventQueueSize: "0"
```

---

got it